.github/workflows/update-docs-version.yml

```diff
@@ -9,21 +9,18 @@ jobs:
         runs-on: ubuntu-18.04
         steps:
             -   uses: actions/checkout@v1
+            -   name: Checkout master
+                run: git checkout master
             -   name: Update latest_version property in mkdocs.yml
                 run: |
                     sed -i "s/latest_version: .*/latest_version: ${GITHUB_REF##*/}/g" mkdocs.yml
+                    git diff
             -   name: Commit changes
-                env:
-                    COMMIT_MSG: |
-                        Update documentation version to ${GITHUB_REF##*/}
-                        skip-checks: true
                 run: |
                     git config user.email "nobody@testcontainers.org"
                     git config user.name "GitHub Action - Update docs version"
-                    git checkout master
-                    git add .
-                    # Only commit and push if we have changes
-                    git diff && git diff --staged --quiet || (git commit -m "${COMMIT_MSG}")
+                    git add mkdocs.yml
+                    git commit -m "Update documentation version to ${GITHUB_REF##*/}" -m "skip-checks: true" || echo "not changed"
```

Comment:
double (`-m`) message?

Reply:
Interestingly, it’s both valid and git CLI’s way to create a multi-line commit message. It was a learning for me, but it does what we need, without the faff of escaping a `\n` in the string 😄